base/shell/rshell/misc.cpp

```diff
@@ -274,3 +274,11 @@ LPVOID *ppv)
 
     return ShellObjectCreatorInit<CRShellClassFactory>(rclsid, riid, ppv);
 }
+
+VOID WINAPI ShowFolderOptionsDialog(UINT Page, BOOL Async)
```

Comment:
What about `Async`?

Reply:
> What about `Async`? And shouldn't this be exported?

CDesktopBrowser::OnShowOptionsDlg when implemented in rshell uses rundll32 to call the real implementation in shell32. The function is never async when called by rundll32 which is why the rshell implementation ignores that parameter. From the callers point of view it is async however because of the punting to rundll32.

It is not exported because shell32::Options_RunDLL is the export Windows (and ROS) has.